StRoot/StMuDSTMaker/COMMON/StMuFstRawHit.h

```diff
@@ -67,7 +67,7 @@ class StMuFstRawHit : public TObject {
     UChar_t     mMaxTimeBin;                ///< the max ADC time bin index of the raw hit
     UShort_t    mIdTruth;                   ///< for embedding, 0 as background
 
-    static UChar_t mDefaultTimeBin;
+    UChar_t     mDefaultTimeBin;
```

Comment:
When I commented about the static member in a persistent class I actually meant to remove it completely or at least remove it from the streamer. Here is how you can tell ROOT not to stream a data member:

```diff
-    static UChar_t mDefaultTimeBin;
+    static UChar_t mDefaultTimeBin;   //!
```

Reply:
Thanks for clarification. I have changed back the codes and add the `//!` as you suggested.